src/main/java/com/thebois/models/beings/AbstractBeing.java

```diff
@@ -153,8 +160,40 @@ protected void setPath(final Collection<Position> path) {
         return Optional.of(path.peek().deepClone());
     }
 
+    protected Position getFinalDestination() {
+        if (path.isEmpty()) return null;
+        return path.firstElement().deepClone();
+    }
+
```

Comment:
Question

As usual I might be missing something obious, but what is the use case for this? Just wondering so that I get a better understand of the code 😄

Reply:
AbstractBeing has another method called getDestination which is for getting the next position in the path, finalDestination is the absolut last position in the path. This will probably be moved to actions later